frontend/src/views/container/container/operate/index.vue

```diff
@@ -788,6 +777,18 @@ const isFromApp = (rowData: Container.ContainerHelper) => {
     }
     return false;
 };
+
+const splitWithQuotes = (str) => {
+    str = str.replace(/\\"/g, '<quota>');
+    const regex = /(?=(?:[^'"]|['"][^'"]*['"])*$)\s+/g;
+    let parts = str.split(regex).filter(Boolean);
+    let returnList = [];
+    for (const item of parts) {
+        returnList.push(item.replaceAll('<quota>', '\\"'));
+    }
+    return returnList;
+};
+
 onMounted(() => {
     if (router.currentRoute.value.query.containerID) {
         isCreate.value = false;
```

Comment:
This code snippet doesn't have any regular deviations mentioned that aren't expected with modern Vue development practices but may not necessarily lead to performance concerns since it's highly unlikely to exceed current usage levels.

However, for optimizations, you could consider:

- Using computed properties instead of using `$ref` directly when creating components (`Command`, `Confirm`).
  
Additionally, ensure all elements like `i18n.global.t` are being used correctly. For instance, `useI18n()`
could be an option here.

Remember the key thing with Vue is consistency — everything should be clear, modular, and properly integrated into each component so that changes don’t cascade throughout your app without affecting other parts too.
It would also help to keep track of how many times this script gets executed because side effects can cause noticeable delays. Consider re-running checks at specific intervals to make sure nothing went wrong between updates.